companies/migrations/0001_initial.py

```diff
@@ -0,0 +1,43 @@
+# Generated by Django 4.1 on 2024-02-13 17:44
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Companies',
+            fields=[
+                ('company_id', models.BigAutoField(primary_key=True, serialize=False)),
+                ('brand', models.CharField(blank=True, max_length=255)),
+                ('is_registered', models.BooleanField(default=False)),
+                ('common_info', models.TextField(blank=True)),
+                ('contact_phone', models.CharField(blank=True, max_length=255)),
+                ('contact_email', models.CharField(blank=True, max_length=255)),
+                ('registration_date', models.DateTimeField(auto_now_add=True)),
+                ('edrpou', models.IntegerField(blank=True)),
+                ('address', models.TextField(blank=True, max_length=255)),
+                ('product_info', models.TextField(blank=True)),
+                ('startup_idea', models.TextField(blank=True)),
+                ('tags', models.CharField(blank=True, max_length=255)),
+            ],
+        ),
+        migrations.CreateModel(
+            name='CompaniesAndUsersRelations',
+            fields=[
+                ('relation_id', models.BigAutoField(primary_key=True, serialize=False)),
+                ('position', models.IntegerField()),
```

Comment:
The 'position' field is an IntegerField without further constraints; if this field represents a predefined set of values, consider using models.Choices for clarity

Reply:
`class CompanyAndUserRelation(models.Model):

    HEAD = "H"
    REPRESENTATIVE = "R"

    POSITION_CHOICES = ((HEAD, "Head"), 
                        (REPRESENTATIVE, "Representative"))

    relation_id = models.BigAutoField(primary_key=True)
    user_id = models.ForeignKey(CustomUser, on_delete=models.CASCADE)
    company_id = models.ForeignKey(Company, on_delete=models.CASCADE)
    position = models.CharField(default=REPRESENTATIVE, choices=POSITION_CHOICES, blank=False, null=False)`